### PR TITLE
Receptive fields and retinotopic mapping

### PR DESCRIPTION
**Pull request by UlrikeS91**
related to #227 

I'm not sure my definitions are 100% accurate but this is what I could find out about the techniques. One major note is that RF mapping can be performed for other sensory areas as well - not only for visual RFs. 

@tgbugs and @lzehl

### CODE REVIEW

File: instances/technique/receptiveFieldMapping.jsonld [hunk 1]
```diff
@@ -4,11 +4,11 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/technique/receptiveFieldMapping",
   "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
-  "definition": null,
+  "definition": "'Receptive field mapping' is use to map out which portion of sensory space elicit neuronal responses when stimulated.",
```
Comment (lzehl) [inline instances/technique/receptiveFieldMapping.jsonld]: **Alternative definition:**  "'Receptive field mapping' is a stimulation technique used to map out which neuron or neuron population preferably elicit a neuronal response to a specific sensory stimulus or sensory stimulus set." 

@tgbugs I think we need your additional feedback here for a nice definition.

File: instances/technique/retinotopicMapping.jsonld [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/retinotopicMapping",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": "'Retinotopic mapping' is a special type of receptive field mapping where the visual receptive field position is mapped in retinotopic coorinates in a brain region.",
```
Comment (lzehl) [inline instances/technique/retinotopicMapping.jsonld]: Better definition: "'Retinotopic mapping' is a stimulation technique in which neurons or neuron populations within the visual stream are mapped point-to-point to visual input of the retinal surface.

@tgbugs also here I think we need you feedback for a nice definition.

**Review by lzehl — CHANGES_REQUESTED** [review]
@tgbugs we need your help with the definitions :)

**Comment by lzehl** [comment]
@UlrikeS91 I've made some changes in the definitions. Please double check. If you are okay with the changes I suggest we merge this one. Corrections of the definitions by @tgbugs can be integrated later (if necessary).

**Review by UlrikeS91 — COMMENTED** [review]
I did a very minor update to the definition of the RF mapping (a "to" was missing) and added "RF mapping" as synonym. 

I can't approve my own PR (therefore I only comment). But I think the changes are good and we can merge now.

**Review by lzehl — APPROVED** [review]

**Pull request merged**